source/fundamentals/linq.txt

```diff
@@ -497,6 +497,82 @@ The following is a subset of the returned results:
        ]
     }
 
+$vectorSearch
+~~~~~~~~~~~~~
+
+The ``$vectorSearch`` aggregation stage performs an *approximate nearest neighbor* search
+on an Atlas Vector Search index on the specified field.
+
+.. tip::
+
+   To obtain the sample dataset used in the following example, see :ref:`csharp-quickstart`.
+   To create the sample Atlas Vector Search index used in the following example, see
+   :atlas:`Create an Atlas Vector Search Index </atlas-vector-search/create-index>` in the
+   Atlas manual.
+
+Consider the ``embedded_movies`` collection in the ``sample_mflix`` database. You
+can use a ``$vectorSearch`` stage to perform a semantic search on the ``plot_embedding``
+field of the documents in the collection.
+
+The following ``EmbeddedMovie`` class models the documents in the ``embedded_movies``
+collection:
+
+.. code-block:: csharp
+
+   [BsonIgnoreExtraElements]
+   public class EmbeddedMovie
+   {
+      [BsonIgnoreIfDefault]
+      public string Title { get; set; }
+
+      public string Plot { get; set; }
+
+      [BsonElement("plot_embedding")]
+      public double[] Embedding { get; set; }
+   }
+
+The following example shows how to use LINQ to generate a ``$vectorSearch`` stage to search
```

Comment:
This is not really LINQ.  It's just `IMongoQueryable` is the entry point for VectorSearch stage, as well as for LINQ expressions. But the next `Select` statement is part of LINQ..